docs/assets/style.css

```diff
@@ -0,0 +1,13 @@
+html,
+body {
+  min-height: 100vh;
+}
+
+.footer {
+  position: sticky;
+  top: 100vh;
+}
+
+.border-subtle {
+  --bs-border-color: var(--bs-border-color-subtle);
```

Comment:
Why did you need this class in place of border-light like describe on https://boosted.orange.com/docs/5.3/utilities/borders/ ?

Reply:
You give me the explanation that you wanted to border color to be adapt to the dark mode